integration/config/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+
+shipyard_dir=".e2e-harness/workdir/.shipyard"
+mkdir -p "${shipyard_dir}"
+chmod -R 777 ".e2e-harness"
+kind get kubeconfig --name="kind" | sudo tee "${shipyard_dir}/config" > /dev/null
+
+
+shipyard_dir="/workdir/.shipyard"
+sudo mkdir -p "${shipyard_dir}"
+sudo chmod -R 777 "/workdir"
+kind get kubeconfig --name="kind" > "${shipyard_dir}/config"
+
+curl -L https://get.helm.sh/helm-v2.16.1-linux-amd64.tar.gz > ./helm.tar.gz
+tar xzvf helm.tar.gz
+chmod u+x linux-amd64/helm
+sudo mv linux-amd64/helm /usr/local/bin/
```

Comment:
Installing helm binary should be moved to architect-orb. But it's good to have it here during prototyping.

I'd say that installing tiller should be also moved there because of the RBAC craziness.